warehouse/models/mart/gtfs/_mart_gtfs_dims.yml

```diff
@@ -51,13 +51,35 @@ models:
       Definitions for the original GTFS fields are available at:
       https://gtfs.org/reference/static#agencytxt.
     columns:
-      - name: key
+      - &schedule_dim_pk
+        name: key
         description: |
-          Synthetic primary key constructed from `feed_key` and `agency_id`.
-        tests: &primary_key_tests
+          Synthetic primary key constructed from `feed_key` and `_line_number`. This key is intended to identify
+          an actual _row_ in the GTFS source data, rather than a GTFS entity described in that row. See
+          _gtfs_key as a single key derived from the "natural" key of the entity.
+        tests:
+          - not_null
+          - unique
+      - &schedule_dim_gtfs_key
+        name: _gtfs_key
+        description:
+          Synthetic primary natural key constructed from the model's GTFS specification uniqueness.
+        tests:
+          - not_null
           - unique:
               # TODO: remove this once we deal with Auburn feed that has tab-delimited files
+              # and at that point can move the &primary_key_tests anchor up to here
+              # (until exclusion is dropped, this exclusion condition is different because it's for key instead of feed_key)
               where: "feed_key != '6368fe701bdd68c4f521751a9a222a10'"
+      - &schedule_dim_dt
+        name: _dt
+        description: Date of the feed version; used for upstream partition elimination.
```

Comment:
Could add "corresponds to date extracted from `_feed_valid_from`" or note that it's a UTC date not a Pacific one